packages/modular-scripts/src/preflight.ts

```diff
@@ -0,0 +1,52 @@
+import { JSONSchemaForNPMPackageJsonFiles as PackageJson } from '@schemastore/package';
+import chalk from 'chalk';
+import execa from 'execa';
+import * as fs from 'fs-extra';
+import path from 'path';
+import updateNotifier from 'update-notifier';
+
+async function isYarnInstalled(): Promise<boolean> {
+  try {
+    await execa('yarnpkg', ['-v']);
+    return true;
+  } catch (err) {
+    return false;
+  }
+}
+
+async function preflight(): Promise<void> {
```

Comment:
could you call this (and the filename) `preflightCheck` instead? preflight is an adjective, not a good function name.